.github/workflows/kunit.yml

```
@@ -32,7 +32,9 @@ jobs:
         run: rustup target add ${{ matrix.target }}
 
       - name: Install qemu
-        run: sudo apt install qemu-system
+        run: |
+          sudo apt update
+          sudo apt install qemu-system
```

```suggestion
        run: |
          sudo apt update
          sudo apt upgrade
          sudo apt install qemu-system
```

Shouldn't the packages that can be updated upgraded as well? Or am I missing something? I believe that `apt update` only retrieves the list of updates without installing any of them

---

> I believe that apt update only retrieves the list of updates

And update the package database. The problem here is because local package database is not up-to-date, apt try to download a version of qemu which is not available anymore (if I remember correctly)

---

Oh, makes sens